examples/migrations/google-adk/artifacts/adk-live-run/memanto-roundtrip-export/sessions/google-adk-okf-atlascraft-review-20260801_2026-07-06_unknown_summary.md

```
@@ -0,0 +1,34 @@
+# Session Summary for google-adk-okf-atlascraft-review-20260801
+**Session ID:** `unknown`
+
+---
+
+### [2026-07-06 09:00:02] [FACT] Project stack
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)